plugin/arcconf/arcconf.py

```diff
@@ -41,17 +42,18 @@
 DRIVE_STATE_UNKNOWN = 20
 
 # Controller states
-CONTROLLER_STATUS_WORKING = 0
+CONTROLLER_STATUS_WORKING, CONTROLLER_STATUS_WORKING_DISPLAY = 0, 'Optimal'
```

Comment:
Can this(and many other lines) split into two lines?

Not a bid deal, just to align with other codes.